### PR TITLE
Steve's updates past v6.3.03 for new tagged version v.6.3.04

### PR DESCRIPTION
**Pull request by smsaleeb**
This update includes Leah's LBC updates + all my INP and cloud nucleation updates. Also, renaming of CCN and GCCN categories. Updated documentation to make it more current. Tagged latest code as v6.3.04.

### CODE REVIEW

File: src/io/rname.f90
Comment (ldgrantcsu) [inline src/io/rname.f90:1]: Is this a change in the namelist settings? Is it documented in the RAMSIN, so CIN_MAX can be set to as low as -2 and that activates the new SPICULE or Peter's MC3E profiles?

**Comment by ldgrantcsu** [comment]
Hey Steve, overall looks good, I have three questions first.
- Did we want to incorporate the other INCUS code changes? Or at a later time?
- We should add namelist options for the new nudging code I added (e.g. timescale for nudging and number of grid points to nudge in each grid) but this could be a to-do for a later time
- Question I posted with the commit on new INP profiles - it looks like the namelist setting range was changed, just wanted to check if that was updated in the namelist description too.

**Comment by smsaleeb** [comment]
I did not put any info on the use of CIN_MAX = -2 or -1 in the namelist
setting since I'm not sure exactly where this will go in the future. But
there is a note in the namelist doc for the user to look at "init_ifn" for
how the profile is initialized. The SPICULE and MC3E profiles are hardcoded
if chosen, such that they don't really use the CIN_MAX value. I was just
using an easy way to trigger these on or off for testing. We can refine
this in the future.

**Comment by smsaleeb** [comment]
Let's incorporate INCUS code changes later. I'll leave that up to you and
the INCUS team.

Perhaps we should add the namelist nudging code flags now.

Could you write up a short blurb I could add to the RAMSIN namelist
document about the names of the nudging code flags, what they do, and how
the values should be chosen? If you can do that, then I'll add the flags to
the code, test them out, and send you an updated pull request.

Thanks,
  Steve

**Comment by smsaleeb** [comment]
Added updated sponge layer flags and documentation. I think this is ready to go.

**Review by ldgrantcsu — APPROVED** [review]
OK thanks Steve for all the feedback and for going ahead and adding the nested grid namelist flags. It looks good!

File: src/io/rname.f90
Comment (ldgrantcsu) [reply src/io/rname.f90:1]: Steve updated documentation and this is commented in the code. Not intended to be available for general use

**Pull request merged**